app/assets/javascripts/adoption_app_home_auth.js

```diff
@@ -0,0 +1,11 @@
+$(document).ready(function () {
+    $(".own-home-auth").hide();
+    $(".residence-auth-pop").change(function () {
+        if (this.value == 'own') {
+            $(".own-home-auth").show();
+        }
+        else if (this.value == 'rent') {
+            $(".own-home-auth").hide();
+        }
```

Comment:
should this just be an else hide?  otherwise it's going to keep whatever state it's in unless the value is own or rent